### ISSUE TITLE
Estimated Total Size (MB）

### ISSUE THREAD

**Issue by HYD-PG** [TylerYep/torchinfo#353]
Thank you very much for providing such a useful tool. When I was using it to test the Transformer model, I wanted to obtain the **Estimated Total Size (MB)**. However, the terminal displayed the following messages without showing the result.
`======================================================================
Layer (type:depth-idx)                        Param #
======================================================================
Model                                         --
├─DataEmbedding: 1-1                          --
│    └─TokenEmbedding: 2-1                    --
│    │    └─Conv1d: 3-1                       493,056
│    └─PositionalEmbedding: 2-2               --
│    └─TimeFeatureEmbedding: 2-3              --
│    │    └─Linear: 3-2                       2,048
│    └─Dropout: 2-4                           --
├─Encoder: 1-2                                --
│    └─ModuleList: 2-5                        --
│    │    └─EncoderLayer: 3-3                 1,577,984
│    │    └─EncoderLayer: 3-4                 1,577,984
│    │    └─EncoderLayer: 3-5                 1,577,984
│    │    └─EncoderLayer: 3-6                 1,577,984
│    └─LayerNorm: 2-6                         1,024
├─DataEmbedding: 1-3                          --
│    └─TokenEmbedding: 2-7                    --
│    │    └─Conv1d: 3-7                       493,056
│    └─PositionalEmbedding: 2-8               --
│    └─TimeFeatureEmbedding: 2-9              --
│    │    └─Linear: 3-8                       2,048
│    └─Dropout: 2-10                          --
├─Decoder: 1-4                                --
│    └─ModuleList: 2-11                       --
│    │    └─DecoderLayer: 3-9                 2,629,632
│    └─LayerNorm: 2-12                        1,024
│    └─Linear: 2-13                           164,673
======================================================================
Total params: 10,098,497
Trainable params: 10,098,497
Non-trainable params: 0
======================================================================
[INFO] Register count_convNd() for <class 'torch.nn.modules.conv.Conv1d'>.
[INFO] Register count_linear() for <class 'torch.nn.modules.linear.Linear'>.
[INFO] Register zero_ops() for <class 'torch.nn.modules.dropout.Dropout'>.
[INFO] Register count_normalization() for <class 'torch.nn.modules.normalization.LayerNorm'>.`
Here is the program I ran:
`summary(self.model, inputs=(batch_x, batch_x_mark, dec_inp, batch_y_mark))`

**Comment by TylerYep** [comment]
`inputs` is not a valid parameter to `torchinfo.summary(...)`
